securedrop_client/gui/widgets.py

```diff
@@ -3589,18 +4048,38 @@ class DeleteSourceAction(QAction):
     def __init__(self, source, parent, controller):
         self.source = source
         self.controller = controller
-        self.text = _("Delete source account")
+        self.text = _("Entire source account")
 
         super().__init__(self.text, parent)
 
-        self.messagebox = DeleteSourceMessageBox(self.source, self.controller)
+        self.confirmation_dialog = DeleteSourceDialog(self.source, self.controller)
         self.triggered.connect(self.trigger)
 
     def trigger(self):
         if self.controller.api is None:
             self.controller.on_action_requiring_login()
         else:
-            self.messagebox.launch()
+            self.confirmation_dialog.exec()
+
+
+class DeleteConversationAction(QAction):
+    """Use this action to delete a source's submissions and replies."""
+
+    def __init__(self, source, parent, controller):
+        self.source = source
+        self.controller = controller
+        self.text = _("Files and messages")
+
+        super().__init__(self.text, parent)
+
+        self.confirmation_dialog = DeleteConversationDialog(self.source, self.controller)
+        self.triggered.connect(self.trigger)
```

Comment:
it's difficult to find in this diff but where is `triggered` is defined?

Reply:
Oh I see, it's a `QAction` signal. This is fine then. Could add a code comment for clarity but we don't do this for other builtin signals like `clicked` so it's fine. Didn't take more than a few seconds to figure out.

Reply:
It's a signal of QAction: https://doc.qt.io/qt-5/qaction.html#signals